TUnit.Core/Context.cs

```diff
@@ -14,21 +14,21 @@ TestContext.Current as Context
         ?? BeforeTestDiscoveryContext.Current as Context
         ?? GlobalContext.Current;
     
-    public StringWriter OutputWriter { get; } = new();
-    public StringWriter ErrorOutputWriter { get; } = new();
+    public StringWriter OutputWriter { get; } = new ThreadSafeStringWriter();
```

Comment:
Does `TextWriter.Synchronized(new StringBuilder())` work? The type would need to be changed to `TextWriter` though.

Reply:
Ooh I didn't know about that. That makes life easier.